common/src/main/java/com/wynntils/features/user/ChatShowRealNameFeature.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.handlers.chat.event.ChatMessageReceivedEvent;
+import java.util.List;
+import net.minecraft.ChatFormatting;
+import net.minecraft.network.chat.Component;
+import net.minecraft.network.chat.HoverEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class ChatShowRealNameFeature extends UserFeature {
+    // credits to avomod for part of the code
+    private static Component eventOriginalMessage;
+    private static String eventOriginalColorMessage;
+
+    @SubscribeEvent
+    public void onChatMessage(ChatMessageReceivedEvent event) {
+        eventOriginalMessage = event.getOriginalMessage();
+        eventOriginalColorMessage = event.getOriginalCodedMessage();
+        System.out.println("Original Coloured message: " + event.getOriginalCodedMessage());
+        if (event.getOriginalCodedMessage().contains("§3[")) {
+            for (Component siblingMessage : event.getOriginalMessage().getSiblings()) {
+                System.out.println("Sibling message: " + siblingMessage);
```

Comment:
What's this? Just tons of debug output left. This code is nowhere near ready to be merged. Please set this PR as draft or withdraw it until it is ready to review.